server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VariableDefinitionDelegate.java

```diff
@@ -345,6 +387,13 @@ private void addError(String suggestion, Locality locality, ErrorSeverity severi
         format("Syntax error defined by %s: %s", getClass().getSimpleName(), error.toString()));
   }
 
+  private void checkRedefinesContainsValue(VariableDefinitionContext variable) {
+    if (!variable.getRedefinesClauses().isEmpty() && !variable.getValueClauses().isEmpty()) {
```

Comment:
Better to get rid-off  as-much-as negations, this one is same condition: 
`if (!(variable.getRedefinesClauses().isEmpty() || variable.getValueClauses().isEmpty())) `